### ISSUE TITLE
Sometimes the server process doesnt close and throws this error.

### ISSUE THREAD

**Issue by Synkstar** [storm37000/gmsv_antifreeze#3]
```
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
```

**Comment by storm37000** [comment]
It always stops for me, make sure your not launching with -debug and it being stuck doing the cat command (you can check this with the tree view in htop).

**Comment by Synkstar** [comment]
Yeah idk its strange. I switched the throw std::exception() part out with just std::terminate() and it seems to work fine now. I dont really know c++ very well wish I did because there is some stuff I want to do with it personally. Well with like garrysmod_common and the sourcesdk as well.

**Comment by storm37000** [comment]
Sorry, I have been busy as of late.  I think I did an exception throw so that things shut down more gracefully as a terminate is a hard kill.  I could implement it to do a terminate if the previous stop method fails after some time.  Or I can look into inducing a segfault instead.

**Comment by storm37000** [comment]
Close the issue if the current code is properly closing it for you.
